docs/interfaces.md

```diff
@@ -12,184 +12,350 @@ Slots may support multiple connections to plugs.  For example the OS snap
 exposes the ``network`` slot and all applications that can talk over the
 network connect their plugs there.
 
-## Supported Interfaces - Basic
+The availability of an interface depends on whether snapd is running on a
+classic (eg, traditional desktop or server) or on a native system. Interfaces
```

Comment:
Again, it's not classic vs. native. It's just that interfaces may be available or not for any given system.

I don't know why camera and optical-drive are classic only either. They shouldn't be.